app/experimenter/visualization/api/v3/views.py

```diff
@@ -36,6 +35,8 @@ class Statistic:
 
 
 BRANCH_DATA = "branch_data"
+PRIMARY_METRIC_SUFFIX = "_ever_used"
+PRIMARY_METRICS_SET = set()
```

Comment:
Generally we shouldn't put mutable collections at the module level scope, it can lead to really wonky behaviour in Python since they persist between invocations.  Should probably move this into some function scope and pass it around?